### ISSUE TITLE
Site Creation: Rotating the Verticals screen loses input focus

### ISSUE THREAD

**Issue by oguzkocer** [wordpress-mobile/WordPress-Android#8752]
While editing the search input text in the new verticals screen if the device is rotated the focus seems to be lost. Here is a gif showing the issue: https://cloudup.com/c3pprY3xRNo

I believe the domains screen has some custom logic for this. We could have a look at that for inspiration.

**Comment by malinajirka** [comment]
@oguzkocer Either I'm missing the issue or the gif is missing the state after rotation. However, I've tested it and I think the view has focus even after rotation, but the keyboard is hidden.

**Comment by oguzkocer** [comment]
> I think the view has focus even after rotation, but the keyboard is hidden.

That's what I meant by the input focus, I thought the gif was clear enough to explain the issue since the keyboard is hidden after a rotation.

**Comment by oguzkocer** [comment]
This is also the case for the new domains screen I have just finished. It's built similarly to the verticals screen, so I am leaving it to this issue so we can fix it together.

**Comment by oguzkocer** [comment]
I have looked into this quite a bit and as @malinajirka the focus is kept after rotations, but the keyboard is hidden. I've tried to fix it but I couldn't find a way that made sense to me. We could keep track of whether the keyboard is open and re-open it on rotation, but that feels too hacky to me. I've then looked into the other parts of the app and found that this behavior was consistent with them.

The best solution I could come up with was to always open the keyboard on fragments that have the `EditText` (verticals, site info and domains) but then if the keyboard was invisible we'd open it for no reason. We could maybe try to manage it from the activity and the main VM, but again, this feels wrong to me.

Since I have opened this issue mostly due to my own misunderstanding of how it all should work, I am going to close it, but if we end up wanting to improve this behavior I wanted to note of my research. /cc @malinajirka in case I am missing something obvious :)
